assignment2.txt

```diff
@@ -0,0 +1,14 @@
+Questions for assignment 2
+
+1. Yes they can if they are separated by && (and) or || (or).
```

Comment:
Could have used a more thorough answer here, but you're right that any number of operators can be used if they're separated by the && or || operators.